### ISSUE TITLE
feature request: allow to use a distant storage 

### ISSUE THREAD

**Issue by klorydryk** [YunoHost-Apps/borg_ynh#34]
I'm using a NAS to backup my Yunohost. As it is not possible to install Borg on it, I mount the directory before to start backup, then unmount.
I took the code from https://borgbackup.readthedocs.io/en/stable/quickstart.html :
```
$ sshfs user@hostname:/path/to /path/to
$ borg init /path/to/repo
$ fusermount -u /path/to
```
It is possible to add this ability to mount a distant storage to your tool ?

**Comment by zamentur** [comment]
It should be done in the PR to integrate that in core.

Until this release, you could do it by installing borserver on your yunohost itself and use a prebackup hook to mount the sshfs into /home/USER/backup.

**Comment by zamentur** [comment]
Partially fixed with the new local repo support in #71
